src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -392,9 +369,96 @@ namespace Utils
             m_db->CompactRange(compactOptions, nullptr, nullptr);
         }
 
+        /**
+         * @brief Initialize transaction.
+         * @return RocksDBTransaction Transaction object.
+         */
+        std::unique_ptr<RocksDBTransaction> createTransaction()
+        {
+            return std::make_unique<RocksDBTransaction>(this);
+        }
+
+        void commit() override
```

Comment:
This method appears to be called in `dabaseFeedManager.hpp:157`, but the behavior is undefined. Is the expected behavior?

Reply:
yes, because I use an abstract base class, and this method is needed when the IRocksDBWrapper points to RockDBTransacion